.ci/jenkins/ci.groovy

```diff
@@ -0,0 +1,129 @@
+                                --container_name=${configMap["dockerContainerName"]} \
+                                --volumes="-v ${WORKSPACE}/${BUILD_NUMBER}:/logs -v ${HOME}/ONNX_CI:/home -v ${WORKDIR}:/root"
+        """
+    }
+    UTILS.CreateStage("Run_docker_containers", runContainerMethod, configurationMaps)
+}
+
+def BuildNgraph(configurationMaps) {
+    Closure buildNgraphMethod = { configMap ->
+        UTILS.PropagateStatus("Run_docker_containers", configMap["dockerContainerName"])
+        sh """
+            docker cp ${CI_ROOT}/update_ngraph.sh ${configMap["dockerContainerName"]}:/home
+            docker exec ${configMap["dockerContainerName"]} ./home/update_ngraph.sh
+        """
+    }
+    UTILS.CreateStage("Build_NGraph", buildNgraphMethod, configurationMaps)
+}
+
+def RunToxTests(configurationMaps) {
+    Closure runToxTestsMethod = { configMap ->
+        UTILS.PropagateStatus("Build_NGraph", configMap["dockerContainerName"])
+        sh """
+            NGRAPH_WHL=\$(docker exec ${configMap["dockerContainerName"]} find /home/ngraph/python/dist/ -name 'ngraph*.whl')
+            docker exec -e TOX_INSTALL_NGRAPH_FROM=\${NGRAPH_WHL} ${configMap["dockerContainerName"]} tox -c /root
+        """
+    }
+    UTILS.CreateStage("Run_tox_tests", runToxTestsMethod, configurationMaps)
+}
+
+def Cleanup(configurationMaps) {
+    Closure cleanupMethod = { configMap ->
+        sh """
+            ${CI_ROOT}/utils/docker.sh chmod --container_name=${configMap["dockerContainerName"]} --directory="/logs" --options="-R 777" || true
+            ${CI_ROOT}/utils/docker.sh stop --container_name=${configMap["dockerContainerName"]} || true
+            ${CI_ROOT}/utils/docker.sh remove --container_name=${configMap["dockerContainerName"]} || true
+            ${CI_ROOT}/utils/docker.sh clean_up || true
+        """
+    }
+    UTILS.CreateStage("Cleanup", cleanupMethod, configurationMaps)
+}
+
+def Notify() {
+    configurationMaps = []
+    configurationMaps.add([
+        "name": "notify"
+    ])
+    Closure notifyMethod = { configMap ->
+        UTILS.PropagateStatus("Cleanup", configMap["dockerContainerName"])
+        if(currentBuild.result != "FAILURE") {
+            currentBuild.result = "SUCCESS"
+        }
+        emailext (
+            subject: "NGraph-Onnx CI: PR $ghprbPullId",
+            body: """
+                <table style="width:100%">
+                    <tr><td>Status:</td> <td>${currentBuild.result}</td></tr>
+                    <tr><td>Repository</td> <td>$ghprbGhRepository</td></tr>
+                    <tr><td>Branch:</td> <td>$ghprbSourceBranch</td></tr>
+                    <tr><td>Pull Request:</td> <td>$ghprbPullId</td></tr>
+                    <tr><td>Commit SHA:</td> <td>$ghprbActualCommit</td></tr>
+                    <tr><td>Link:</td> <td>$ghprbPullLink</td></tr>
+                </table>
+            """,
+            to: "$ghprbPullAuthorEmail"
```

Comment:
It may be a good idea to send the email to both people. The PR owner will want to know, but the commiter will also want to know to fix the issue. It's not uncommon for 2 people to work on one PR.